flake.nix

```diff
@@ -74,6 +74,11 @@
             '';
           };
 
+          # The casper-client's build.rs requires git at buildtime
+          git-mock = pkgs.writeShellScriptBin "git" ''
+            echo "got ya"
```

Comment:
?

Reply:
https://github.com/casper-ecosystem/casper-client-rs/blob/6b61f160a536fafbfb541b1ecb20d05a8e666fd3/build.rs#L7-L12
The comment above explains, maybe I should add the link to the comment. I don't want to make git a build time dependency.

Reply:
Is `echo "got ya"` necessary?

Reply:
It can be anything, do you have any preference? I can make it simply exit with a success status. The return value is not used in any significant way.

Reply:
Ah okay, got it.